backup_and_restore/application_backup_and_restore/troubleshooting/oadp-restic-issues-doc.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: CONCEPT
+[id="oadp-restic-issues-doc"]
+= Restic issues
+include::_attributes/common-attributes.adoc[]
+:context: oadp-troubleshooting
+
+You might encounter these issues when you back up applications with Restic.
+
+[id="restic-permission-error-nfs-root-squash-enabled-doc"]
+== Restic permission error for NFS data volumes with root_squash enabled
+
+The `Restic` pod log displays the error message: `controller=pod-volume-backup error="fork/exec/usr/bin/restic: permission denied"`.
+
+.Cause
+
+If your NFS data volumes have `root_squash` enabled, `Restic` maps to `nfsnobody` and does not have permission to create backups.
+
+.Solution
+
+You can resolve this issue by creating a supplemental group for `Restic` and adding the group ID to the `DataProtectionApplication` manifest:
+
+. Create a supplemental group for `Restic` on the NFS data volume.
+. Set the `setgid` bit on the NFS directories so that group ownership is inherited.
+. Add the `spec.configuration.restic.supplementalGroups` parameter and the group ID to the `DataProtectionApplication` manifest, as in the following example:
++
+[source,yaml]
+----
+spec:
+  configuration:
+    restic:
+      enable: true
+      supplementalGroups:
+      - <group_id> <1>
+----
+<1> Specify the supplemental group ID.
+
+. Wait for the `Restic` pods to restart so that the changes are applied.
+
+[id="restic-backup-cannot-be-recreated-after-s3-bucket-emptied-doc"]
+== Restic Backup CR cannot be recreated after bucket is emptied
+
+If you create a Restic `Backup` CR for a namespace, empty the object storage bucket, and then recreate the `Backup` CR for the same namespace, the recreated `Backup` CR fails.
+
+The `velero` pod log displays the following error message: `stderr=Fatal: unable to open config file: Stat: The specified key does not exist.\nIs there a repository at the following location?`.
+
+.Cause
+
+Velero does not recreate or update the Restic repository from the `ResticRepository` manifest if the Restic directories are deleted from object storage. See link:https://github.com/vmware-tanzu/velero/issues/4421[Velero issue 4421] for more information.
+
+.Solution
+
+* Remove the related Restic repository from the namespace by running the following command:
```

Comment:
In 1.2 releases, there is no more resticrepository CRD, the new CRD name is: BackupRepository.

Reply:
Please see new addition - https://github.com/openshift/openshift-docs/pull/62431/commits/97372c5c56b0cff194c4a5de106ecc15a1a596ae